lib/response.js

```diff
@@ -55,6 +55,22 @@ module.exports = res
 
 var charsetRegExp = /;\s*charset\s*=/;
 
+/**
+ * Helper to check if status code is deprecated
+ *
+ * @param {Number | String} code
+ * @return {Boolean}
+ * @private
+ */
+function isStatusCodeDeprecated(code) {
+  if (code > 99 && code < 1000) {
```

Comment:
Sorry if this have been mentioned, just read thru the discussion twice and couldn't find it: What is the reason for only deprecating `code > 99 && code < 1000`? e.g. `isStatusCodeDeprecated('hello')` returns `false` now, but I believe that that will throw in 5.x?

Reply:
The logic here is that Node.js would throw on that already. I targeted the dep notices only at cases which Node.js itself would allow previously, but Express will now throw on. Essentially deprecating things you were allowed to do before by Node.js, but that Express will now prevent you from doing.

Reply:
Ahh, thank you for the detailed explanation! ❤️